DevelopGitHub/GitHubExtension/GitHubExtension.Security.DAL/Package/DALSecurityPackage.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace GitHubExtension.Security.DAL.Package
 {
-    public class DALPackage : IPackage
+    public class DALSecurityPackage : IPackage
```

Comment:
Ohh..my. Please do't use DAL PREFIX!!! It's not make sence. Is it SecurityPackage sounds bad?